### DISCUSSION TITLE
Problem with catalog installation

### DISCUSSION THREAD

**Discussion by Kvinki112** [webdriverio/webdriverio#11401] [Technical Discussion]
Hello!
There was a problem with installing the WebdriverIO setting. When running the command "npm init wdio . -- --yes" command gives the following error
Ups, something went wrong: Error calling: npx wdio config!
npm ERR! code 1
npm ERR! path C:\Users\User\wdio-tests
npm ERR! command failed
npm ERR! command C:\Windows\system32\cmd.exe /d /s /c create-wdio ./path/to/new/project

npm ERR! A complete log of this run can be found in: C:\Users\User\AppData\Local\npm-cache_logs\2023-10-11T10_49_07_111Z-debug-0.log
PS C:\Users\User\wdio-tests> 

Here is a screenshot
![image](https://github.com/webdriverio/webdriverio/assets/139227502/f1368fcd-4a67-4170-8bd4-d1035913b519)

**Comment by christian-bromann** [comment]
Please see https://github.com/webdriverio/webdriverio/issues/11399
